### PR TITLE
Remove outdated note that UI trim does not support RWX.

### PR DESCRIPTION
**Pull request by james-munson**
https://github.com/longhorn/longhorn/issues/6733

Revise note that UI does not support trim on RWX volumes.  Duplicate the change to releases 1.5.1-1.5.4

### CODE REVIEW

**Review by PhanLe1010 — APPROVED** [review]
In general LGTM

File: content/docs/1.5.1/volumes-and-nodes/trim-filesystem.md [hunk 1]
```diff
@@ -18,17 +18,33 @@ Since v1.4.0, Longhorn supports trimming filesystem inside Longhorn volumes. Tri
...
 
 Then Longhorn will **try its best** to figure out the mount point and execute `fstrim <the mount point>`.  If something is wrong or the filesystem does not exist, the UI will return an error.
 
-#### Via cmd
+#### Via shell command
 
-Users need to figure out the mount point of the volume then manually execute `fstrim <the mount point>`.
+Users need to figure out the mount point of the volume and then manually execute `fstrim <the mount point>`.  For RWO volumes that would be a pod of its workload, or perhaps the node to which it was manually attached.
+
+For an RWX volume, that would be the share manager pod for that volume, which actually contains the NFS server.  The share manager pod is typically named `share-manager-<volume name>`.  The sequence would be:
```
Comment (jillian-maroket) [inline content/docs/1.5.1/volumes-and-nodes/trim-filesystem.md:33]:
```suggestion
When using `fstrim`, you must identify the mount point of the volume and then run the command `fstrim <the mount point>`.

- NWO volume: The mount point is either a pod of the workload or the node to which the volume was manually attached.
- RWX volume: The mount point is the share manager pod of the volume. The share manager pod contains the NFS server and is typically named `share-manager-<volume name>`.
```

File: content/docs/1.5.1/volumes-and-nodes/trim-filesystem.md [hunk 2]
```diff
@@ -18,17 +18,33 @@ Since v1.4.0, Longhorn supports trimming filesystem inside Longhorn volumes. Tri
...
+
+1. Figure out and enter into the share manager pod of the RWX volume.
```
Comment (jillian-maroket) [inline content/docs/1.5.1/volumes-and-nodes/trim-filesystem.md:35]:
```suggestion
To trim an RWX volume, perform the following steps:

1. Identify and then open a shell inside the share manager pod of the volume.
```

File: content/docs/1.5.1/volumes-and-nodes/trim-filesystem.md [hunk 3]
```diff
@@ -18,17 +18,33 @@ Since v1.4.0, Longhorn supports trimming filesystem inside Longhorn volumes. Tri
...
+    ```
+    kubectl -n longhorn-system exec -it <the share manager pod> -- bash
+    ```
+2. Figure out the work directory of the NFS server. The work directory is typically like `/export/<volume name>`:
```
Comment (jillian-maroket) [inline content/docs/1.5.1/volumes-and-nodes/trim-filesystem.md:39]:
```suggestion
1. Identify the work directory of the NFS server (for example, `/export/<volume name>`).
```

File: content/docs/1.5.1/volumes-and-nodes/trim-filesystem.md [hunk 4]
```diff
@@ -18,17 +18,33 @@ Since v1.4.0, Longhorn supports trimming filesystem inside Longhorn volumes. Tri
...
+    ```
+    mount | grep <volume name>
+    /dev/longhorn/<volume name> on /export/<volume name> type ext4 (rw,relatime)
+    ```
+3. Trim the work directory
```
Comment (jillian-maroket) [inline content/docs/1.5.1/volumes-and-nodes/trim-filesystem.md:44]:
```suggestion
1. Trim the work directory.
```

**Review by jillian-maroket — COMMENTED** [review]
@james-munson Let me know if you have concerns about the changes.

**Comment by james-munson** [comment]
Incorporated the suggestions and duplicated to all changed copies.

**Review by jillian-maroket — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**